src/.npmrc

```diff
@@ -1,4 +1,5 @@
 # pnpm-related options
 shamefully-hoist=true
 strict-peer-dependencies=false
-engine-strict = true
+engine-strict=true
+use-node-version=20.13.1
```

Comment:
I'm open to using a different version here according to your preferences @petkivim!

Reply:
Hi @sschuberth!
Node version is already managed by gradle plugin [here](https://github.com/nordic-institute/X-Road/blob/develop/src/shared-ui/build.gradle#L7), version number is defined in [gradle.properties](https://github.com/nordic-institute/X-Road/blob/develop/src/gradle.properties#L12).

Setting `use-node-version` here would cause it to download again even when using the same version number for gradle plugin.

Reply:
I see. Maintaining the version in two places would not be nice anyway. I'll try to think of another approach to make the [ORT analyzer](https://oss-review-toolkit.org/ort/docs/tools/analyzer) use the correct Node version.